### PR TITLE
Story portfolio

### PR DESCRIPTION
**Pull request by gidspektor**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            <p>I’m now a trainee web developer at Mayden Academy and after my 16 week journey here I would like to go into web development.
+            </p>
+            <p>I want to create cool, clean and creative websites; as I believe simplicity is key to a good user experience. </p>
+        </div>
+    </section>
+    <section id="project" class="pageThree">
+        <div class="showcase">
+            <div class="cards">
+                <div class="projOne">
+                    <a href="#"></a>
```
Comment (charliecog) [inline index.html]: not doing anything - remove

File: index.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                </div>
+                Project one
+            </div>
+            <div class="cards">
+                <div class="projOne">
+                    <a href="#"></a>
```
Comment (charliecog) [inline index.html]: not doing anything - remove

File: index.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                 </div>
+                 Project two
+            </div>
+          <div class="cards">
+                <div class="projOne">
+                    <a href="#"></a>
```
Comment (charliecog) [inline index.html]: not doing anything - remove

File: index.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                </div>
+                Project three
+          </div>
+            <div class="cards">
+                <div class="projOne">
+                    <a href="#"></a>
```
Comment (charliecog) [inline index.html]: not doing anything - remove

File: index.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+                </div>
+                Project four
+            </div>
+            <div class="cards">
+                <div class="projOne">
+                    <a href="#"></a>
```
Comment (charliecog) [inline index.html]: not doing anything - remove

File: index.html [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+                </div>
+                Project five
+            </div>
+            <div class="cards">
+                <div class="projOne">
+                    <a href="#"></a>
```
Comment (charliecog) [inline index.html]: not doing anything - remove

File: index.html [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+            </div>
+        </div>
+    </section>
+    <section id="contact" class="pageFour">
+            <div class="myInfo">
+                <h1>Contact</h1>
+                <p>Give me a call</p>
+                <p>07867511476</p>
+                <p>Send me an email</p>
+                <a class="email" href="gidspektor@gmail.com">gidspektor@gmail.com</a>
```
Comment (charliecog) [inline index.html]: use a mailto
https://www.w3schools.com/html/tryit.asp?filename=tryhtml_mailto

File: style.css [hunk 8]
```diff
@@ -0,0 +1,296 @@
...
+    margin: 0;
+    text-align: center;
+    font-family: Georgia, sans-serif;
+}
+
+nav {
+    width: 100%;
+    height: 80px;
+    position: fixed;
+    top: 0;
```
Comment (charliecog) [inline style.css]: not doing anything

File: style.css [hunk 9]
```diff
@@ -0,0 +1,296 @@
...
+    margin-right: 20%;
+    padding-top: 5px;
+}
+
+.dropdown {
+    float: right;
+    margin-right: 20%;
+    margin-top: 40px;
+    display: none;
+    background-image: url("button.png");
```
Comment (charliecog) [inline style.css]: put your images in an images folder

File: style.css [hunk 10]
```diff
@@ -0,0 +1,296 @@
...
+    display: none;
+}
+
+nav li {
+    display: inline-block;
+    margin-top: 30px;
+}
+
+.navText {
+    font-family: "Arial Rounded MT Bold", sans-serif, "Times New Roman";
```
Comment (charliecog) [inline style.css]: missing "" from sans-serif

File: style.css [hunk 11]
```diff
@@ -0,0 +1,296 @@
...
+
+
+.navText:hover {
+    border-color: #000000;
+}
+
+.pageOne {
+    height: 740px;
+    opacity: 0.6;
+    background-image: url("tree.png");
```
Comment (charliecog) [inline style.css]: images in an image folder

File: style.css [hunk 12]
```diff
@@ -0,0 +1,296 @@
...
+}
+
+.home h2 {
+    font-size: 2.5rem;
+    transition: 0.8s;
+}
+
+.home .button {
+    display: inline-block;
+    background: transparent;
```
Comment (charliecog) [inline style.css]: not doing anything

File: style.css [hunk 13]
```diff
@@ -0,0 +1,296 @@
...
+
+.pageTwo {
+    height: 740px;
+    background-image: radial-gradient(circle, #ffffff, #000000);
+}
+
+.me h1 {
+    font-size: 2.5rem;
+    border-bottom: 3px solid #000000;
+    transition: 0.8s;
```
Comment (charliecog) [inline style.css:142]: transition 0.8s is repeated alot through your code. put it in its own separate class and add to html elements where necessary

**Review by charliecog — CHANGES_REQUESTED** [review]
good stuff, few last changes

File: style.css (hunk 13 above)
Comment (gidspektor) [reply style.css:142]: doesnt work for h1s or other items unless i directly put the transition into their specific css

File: index.html [hunk 14]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <title>Gideon Spektor</title>
+</head>
+<body>
+    <nav>
+        <a href="#"><img src="logo.png"/></a>
```
Comment (charliecog) [inline index.html]: add alt text for image

Comment (charliecog) [reply index.html]: add it for all images

**Pull request merged**